frontend/src/components/Modal/RiderModalInfo.tsx

```diff
@@ -1,112 +1,157 @@
-import React from 'react';
-import { useForm } from 'react-hook-form';
-import cn from 'classnames';
-import { Button, Input, Label } from '../FormElements/FormElements';
-import styles from './ridermodal.module.css';
-import { ObjectType } from '../../types/index';
+import React from "react";
+import { useForm } from "react-hook-form";
+import cn from "classnames";
+import { Button, Input } from "../FormElements/FormElements";
+import styles from "./ridermodal.module.css";
+import { ObjectType, Accessibility } from "../../types/index";
 
 type ModalFormProps = {
   onSubmit: (data: ObjectType) => void;
   formData?: ObjectType;
-}
+};
 
 const RiderModalInfo = ({ onSubmit }: ModalFormProps) => {
-  const { register, errors, handleSubmit } = useForm();
-  const beforeSubmit = ({ name, netid, email, phoneNumber, needs,
-    address, start, end }: ObjectType) => {
-    const startDate = new Date(`${start}`).toISOString();
-    const endDate = new Date(`${end}`).toISOString();
-    const splitName = name.split(' ');
+  const { register, errors, handleSubmit, getValues } = useForm();
+  const beforeSubmit = ({
+    name,
+    netid,
+    phoneNumber,
+    needs,
+    address,
+    joinDate,
+    endDate,
+  }: ObjectType) => {
+    const email = `${netid}@cornell.edu`;
+    const splitName = name.split(" ");
     const firstName = splitName[0];
     const lastName = splitName[1];
-    const accessibilityNeeds = needs.split(',');
+    const accessibility = needs.split(",").map((n: string) => n.trim());
     onSubmit({
-      id: netid,
       firstName,
       lastName,
       email,
       phoneNumber,
-      accessibilityNeeds,
+      accessibility,
       address,
-      startDate,
+      joinDate,
       endDate,
     });
   };
+
   return (
     <form onSubmit={handleSubmit(beforeSubmit)} className={styles.form}>
       <div className={cn(styles.inputContainer, styles.rideTime)}>
-        <div className={cn(styles.gridR1, styles.gridC1)}>
+        <div className={cn(styles.gridR1, styles.gridCSmall1)}>
           <Input
```

Comment:
For all of the inputs you'll need to add some sort of label/identifier. the SRLabel that was created should work in this case. The placeholder text is not sufficient & we'll need some sort of label (even if hidden) to make it accessible. It's tedious but unavoidable. I can give you an example of how to use it if you need.